sale_order_line_price_unit_pricelist/models/sale_order.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    is_price_matched = fields.Boolean(compute="_compute_is_price_matched", store=True)
```

Comment:
We should reverse the objective of the field.

```suggestion
    has_price_discrepancy = fields.Boolean(compute="_compute_has_price_discrepancy", store=True)
```